tests/unit/ops/op_builder/test_rocm.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Tom Rix
```

Comment:
Hi @trixirt - can you please run the formatting check (`pre-commit run --all-files`) - to pass that check on this file?

Reply:
pre-commit errors out with

            File "<string>", line 22, in <module>
            File "/home/trix/.cache/pre-commit/repoyr3bl7h1/yapf/__init__.py", line 35, in <module>
              from yapf.yapflib import errors
            File "/home/trix/.cache/pre-commit/repoyr3bl7h1/yapf/yapflib/errors.py", line 16, in <module>
              from lib2to3.pgen2 import tokenize
          ModuleNotFoundError: No module named 'lib2to3'
          [end of output]
      
      note: This error originates from a subprocess, and is likely not a problem with pip.
    error: subprocess-exited-with-error

Fedora/Rawhide is on python 3.13
The docs on lib2to3 say
[ removed from 3.13
](https://docs.python.org/3/library/2to3.html#module-lib2to3)

Reply:
Running on 3.12 the license check fails because i believe this line 
[ copyright check 
](https://github.com/trixirt/DeepSpeed/blob/master/scripts/check-license.py#L21)

It will pass if I use the Microsoft boilerplate.
There are examples of non Microsoft copyrights in the code, so I wonder how they passed this check.

Reply:
@trixirt - our check requires there to be a license file on top of each file, we can use the standard Microsoft one here since that's most fitting here if you could add it?  Or I can add a dummy test_rocm file with the header as well if that helps.